pkg/utils/helper.go

```diff
@@ -232,3 +234,17 @@ func GetSourceAcctAndArn(roleARN, region, clusterName string) (string, string, e
 	sourceArn := fmt.Sprintf("arn:%s:eks:%s:%s:cluster/%s", parsedArn.Partition, region, parsedArn.AccountID, clusterName)
 	return parsedArn.AccountID, sourceArn, nil
 }
+
+// PodHasENIRequest will return true if first container of pod spec has request for eni indicating
+// it needs trunk interface from vpc-rc
+func PodHasENIRequest(pod *v1.Pod) bool {
+	if pod == nil {
```

Comment:
nit: I don't think pod can be nil here. If it is unfortunately, the nil pointer can panic before this.